README.md

```diff
@@ -34,21 +34,9 @@ write
 - [Animation example](http://wilsonpage.github.io/fastdom/examples/animation.html)
 - [Aspect ratio example](http://wilsonpage.github.io/fastdom/examples/aspect-ratio.html)
 
-## Installation
-
-FastDom is CommonJS and AMD compatible, you can install it in one of the following ways:
-
-``` sh
-$ npm install fastdom
-```
-``` sh
-$ bower install fastdom
-```
-``` sh
-$ component install wilsonpage/fastdom
-```
-or [download](http://github.com/wilsonpage/fastdom/raw/master/index.js).
+## Usage in your repo
 
+Pull in a new version of this dependency into your app repo, edit package.json and either npm update or npm install
```

Comment:
On a fresh new project, devs shouldn't do `npm install fastdom` as it will pull the Wilson version from the Npmjs.com 
Instead they should reference this package as 
```
"fastdom": "github:Financial-Times/fastdom#v2.1.1",
``` 
in their package.json.

And for future runs, either do `npm update` or `npm install`

Reply:
Feel free to suggest alternatives if it isn't ideal or clear to you

Reply:
hmmm can we change package.json to point to our repo? https://github.com/Financial-Times/fastdom/blob/main/package.json#L16 would be changed to Financial-Times/fastdom

Reply:
oh ok, the fastdom from wilsonpage is still in the registry...

Reply:
shouldn't we publish our @financial-times/fastdom then?

Reply:
or keep using [wilsonpage-fastdom](https://github.com/npmcomponent/wilsonpage-fastdom) as there is no difference?

Reply:
Yup, I think you've got it about right! Wilson used to work at FT Labs when it was a separate company and people used to publish npm packages under their own usernames at that point - shared accounts were a pain and there was no npm Organisation support yet. Later, where we wanted to keep making some FT-specific changes, the package was forked into the then-financial-times namespace for easier maintenance.

I don't think it's worth re-publishing to npm as a result - seeing as we've essentially deprecated this, I don't think that gains us much other than a slightly simpler install...?

(The main task was to remove installation via bower, but - the `bower.json` in this case is more of a way to allow this package to be installed via bower still - if this was a published/supported package we'd want to keep that to allow others to still install via bower even though we wouldn't be using that ourselves)

Reply:
thanks both!

Reply:
agree.. or removing bower references would be enough leaving the rest as it is